app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -25,7 +25,6 @@ def send_ready_to_verify_email(user, pii, enrollment)
           UserMailer.in_person_ready_to_verify(
             user,
             email_address,
-            first_name: pii['first_name'],
```

Comment:
Can remove `pii` as a parameter from this function since it's no longer used (and from everywhere the function is called)